WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -147,6 +147,17 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
         ZendeskUtils.sharedInstance.showNewRequestIfPossible(from: sourceViewController, with: sourceTag)
     }
 
+    /// A self-hosted site URL is available and needs validated
+    /// before presenting the username and password view controller.
+    /// - Parameters:
+    ///     - site: passes in the site information to the delegate method.
+    ///     - onCompletion: Closure to be executed on completion.
+    ///
+    func shouldPresentUsernamePasswordController(for siteInfo: WordPressComSiteInfo?, onCompletion: @escaping (Error?, Bool) -> Void) {
+        let isSelfHosted = true
+        onCompletion(nil, isSelfHosted)
```

Comment:
Do we need to declare `isSelfHosted`?  Should we just call `onCompletion(nil, true)`?